pkg/ebpf/tracer.go

```diff
@@ -632,6 +649,16 @@ func readLocalAddresses() map[util.Address]struct{} {
 	return addresses
 }
 
+// getNetworkBlacklists retrieves the blacklists set in the agent config
+func getNetworkBlacklists() *ConnectionBlacklist {
+	s := util.ParseBlacklist(config.Datadog.GetStringMapStringSlice("system_probe_config.excluded_source_connections"))
```

Comment:
Lets not call `config.Datadog.GetStringMapStringSlice` directly. Let's put that code in the config object and parse that.